Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/ToolBar/ToolBarManagerInterface.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/Serialization/SerializeContext.h>
+
+class QToolBar;
+
+namespace AzToolsFramework
+{
+    using ToolBarManagerOperationResult = AZ::Outcome<void, AZStd::string>;
+
+    struct ToolBarProperties
```

Comment:
nit/suggestion: Might be overkill but still useful to state what this is used for